circuit-benchmarks/src/evm_circuit.rs

```diff
@@ -12,7 +13,7 @@ pub struct TestCircuit<F> {
     block: Block<F>,
 }
 
-impl<F: FieldExt> Circuit<F> for TestCircuit<F> {
+impl<F: FieldExt + PrimeField<Repr = [u8; 32]>> Circuit<F> for TestCircuit<F> {
```

Comment:
I discussed with @kilic this and since there are no more `to/from_bytes` methods, `to/from_repr` has to be used.

That means that `F` needs to define it's repr type. And since it's trait bounds, anything that interacts with it needs to have the same bounds which has lead to the propagation of the `PrimeField` trait.

Reply:
I see. Makes sense to me.

Reply:
@ed255 I like your proposal